public/app/features/dashboard/components/PanelEditor/VisualizationSelectPane.tsx

```diff
@@ -28,10 +29,15 @@ interface Props {
 export const VisualizationSelectPane = ({ panel, data }: Props) => {
   const plugin = useSelector(getPanelPluginWithFallback(panel.type));
   const [searchQuery, setSearchQuery] = useState('');
-  const [listMode, setListMode] = useLocalStorage(
-    LS_VISUALIZATION_SELECT_TAB_KEY,
-    VisualizationSelectPaneTab.Visualizations
-  );
+
+  // Add support to show widgets in the visualization picker
+  const isWidget = !!plugin.meta.skipDataQuery;
+  const isWidgetEnabled = !!(isWidget && config.featureToggles.vizAndWidgetSplit);
```

Comment:
Nit for readability: 
```suggestion
  const isWidgetEnabled = Boolean(isWidget && config.featureToggles.vizAndWidgetSplit);
```